tox.ini

```diff
@@ -158,4 +158,4 @@ commands = python -c 'import plasmapy'
 deps =
     mypy >= 1.8.0
 commands =
-    mypy plasmapy --install-types --non-interactive
+    mypy plasmapy --install-types --non-interactive --show-error-context --show-error-code-links --pretty
```

Comment:
We'll probably want to play with these flags to figure out which error output works best.